lib/src/supabase_auth.dart

```diff
@@ -380,13 +381,10 @@ class _OAuthSignInWebViewState extends State<_OAuthSignInWebView> {
 
   FutureOr<NavigationDecision> _handleNavigationRequest(
     NavigationRequest request,
-  ) {
+  ) async {
     if (widget.redirectTo != null &&
         request.url.startsWith(widget.redirectTo!)) {
-      launchUrlString(request.url);
-      if (Navigator.canPop(context)) {
-        Navigator.of(context).pop(true);
-      }
```

Comment:
This entire chunk was unnecessary for the app to relaunch on successful sign in. Have verified that it works on iOS simulator.